apps/web/app/search/page.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import RenderProducts from 'components/search/renderProducts';
+import RenderReviews from 'components/search/renderReviews';
+import { Tabs } from 'components/search/tab';
+import { SEARCH_OPTION } from 'constants/option';
+import { SearchOption } from 'types/option';
+import { useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from '@/components';
+import { SvgHomeFill } from '@/icons';
+import {
+  mockImageReviewSearchResponse,
+  mockProductSearchResponse,
+  mockVideoReviewSearchResponse,
+} from './mockupData';
+
+export default function Page() {
+  const searchParams = useSearchParams();
+  const middleCategory = searchParams.get('middleCategory');
+  const subCategory = searchParams.get('subCategory');
+  const [selectedTab, setSelectedTab] = useState<SearchOption>(
+    SEARCH_OPTION.PRODUCT
+  );
+
+  const productData = mockProductSearchResponse;
+  const reviewVideoData = mockVideoReviewSearchResponse;
+  const reviewImageData = mockImageReviewSearchResponse;
+
+  const handleClickTab = (option: SearchOption) => {
+    setSelectedTab(option);
+  };
+  const handleVideoButton = () => {};
+  const handleImageButton = () => {};
+
+  const TAB_RENDER = {
```

Comment:
```suggestion
  const tabRender = {
```
요건 렌더링할때 사용하니까 대문자 스네이크 대신에 카멜로 해도 좋을 것 같네용